server/routes/ads.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const router = express.Router();
+const { google } = require('googleapis');
+const privatekey = require('../config/client_secret.json');
+const jwtClient = new google.auth.JWT(
+  privatekey.client_email,
+  null,
+  privatekey.private_key,
+  ['https://www.googleapis.com/auth/spreadsheets']
+);
+
+//authenticate request
+jwtClient.authorize(function (err, tokens) {
+  if (err) {
+    console.log(err);
+    return;
```

Comment:
is exiting out the program intentional? is there a use case to throw an error instead?